src/js/pyodide.js

```diff
@@ -38,7 +38,10 @@ globalThis.loadPyodide = async function (config = {}) {
   // Note: PYODIDE_BASE_URL is an environment variable replaced in
   // in this template in the Makefile. It's recommended to always set
   // indexURL in any case.
-  let baseURL = config.indexURL || "{{ PYODIDE_BASE_URL }}";
+  if(!config.indexURL){
+    throw new Error("Undefined indexURL");
```

Comment:
Better to have messages that indicate what the user needs to do to fix it. Maybe something along the lines, 
```suggestion
    throw new Error("Please provide indexURL parameter to loadPyodide");
```

Reply:
Yup I was being lazy.